### ISSUE TITLE
Does the code support generative rm evaluating?

### ISSUE THREAD

**Issue by LittleCoder12345** [Zhou-Zoey/RMB-Reward-Model-Benchmark#5]
Hi, I found that when set the config in eval_config.yaml as 'generative:True', run_rm.sh still use the method of calcing logits to get score, is that normal? I think it should be using prompt to ask the generative model to select A or B. Thanks!
